pkg/operation/hybridbotanist/controlplane.go

```diff
@@ -165,14 +170,33 @@ func (b *HybridBotanist) DeployKubeAPIServer() error {
 
 	apiServerConfig := b.Shoot.Info.Spec.Kubernetes.KubeAPIServer
 	if apiServerConfig != nil {
-		defaultValues["featureGates"] = apiServerConfig.FeatureGates
-		defaultValues["runtimeConfig"] = apiServerConfig.RuntimeConfig
+		featureGates = apiServerConfig.FeatureGates
+		runtimeConfig = apiServerConfig.RuntimeConfig
 
 		if apiServerConfig.OIDCConfig != nil {
 			defaultValues["oidcConfig"] = apiServerConfig.OIDCConfig
 		}
 	}
 
+	geqKubernetes111, err := utils.CheckVersionMeetsConstraint(b.Shoot.Info.Spec.Kubernetes.Version, ">= 1.11")
```

Comment:
Put those checks in and not in go code

https://github.com/gardener/gardener/blob/92ca7287cf2cb7287410c6130fae4b32c9201dd4/charts/seed-controlplane/charts/kube-apiserver/templates/_helpers.tpl#L1-L5

Reply:
https://github.com/gardener/gardener/pull/267/commits/062e57bda20c41dce228a5c4159f28b5eb766de3?utf8=%E2%9C%93&diff=unified#diff-42520e4f41cf1180be16b2734375d5c5R5